content/en/docs/started/workstation/getting-started-multipass.md

```diff
@@ -24,37 +24,61 @@ Here's a summary of the steps involved:
 
 **Note:** the minimum version of Microk8s needed to enable Kubeflow is 1.18
 
-### 1. Install and set up Microk8s
+### 1. Install Microk8s
 
-Run the following commands to install and setup MicroK8s:
+Single-command install MicroK8s snap:
 
 ```
-snap install microk8s --classic
+sudo snap install microk8s --classic
+```
+Confirm install:
+```
 microk8s.status --wait-ready
-# Enable common services:
+```
+
+### 2. Enable Microk8s services:
+
+Enable common services:
+```
 microk8s.enable dns dashboard storage
-# If you have a GPU, run: `microk8s.enable gpu`
 ```
+If you would like to enable GPU passthrough (optional), run: `microk8s.enable gpu`
```

Comment:
I'd keep this short and to the point is good for UX. `If you would like to... (optional), run:` -> `Optional: to enable ... run:`.  Maybe you can use:

```
- Optional: to enable GPU support, run `microk8s.enable gpu`.
```

Also, since it's NVIDIA only, according to the docs, shall this be specified too? (Source: https://microk8s.io/docs/addon-gpu).